routing.rst

```diff
@@ -15,13 +15,23 @@ provides other useful features, like generating SEO-friendly URLs (e.g.
 Creating Routes
 ---------------
 
-Routes can be configured in YAML, XML, PHP or using annotations. All formats
-provide the same features and performance, so choose your favorite.
+Routes can be configured in YAML, XML, PHP or using either attributes or
+annotations. All formats provide the same features and performance, so choose
+your favorite.
 :ref:`Symfony recommends annotations <best-practice-controller-annotations>`
 because it's convenient to put the route and controller in the same place.
 
-Creating Routes as Annotations
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Creating Routes as Attributes or Annotations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On PHP 8, you can use native attributes to configure routes right away. On
+PHP 7, where attributes are not available, you can use annotations instead,
+provided by the Doctrine Annotations library.
+
+.. versionadded:: 5.2
+
+    The ability to use PHP attributes to configure routes was introduced in
+    Symfony 5.2.
```

Comment:
I think this should be moved after the `composer require ...` example + we should reword that paragraph and the next one. It is no longer required to install doctrine annotations if you're using PHP 8 attributes

Reply:
I agree. I was thinking, we should also change the recipes. If `doctrine/annotations` is an optional dependency now, the routes snippet that enables attributes/annotations should not be attached to the `doctrine/annotations` package anymore.

Reply:
That's exactly what I was thinking. The big question is then: what should be the trigger for this recipe? Maybe, we should make it the default routing config when installing in a PHP 8 application? (I don't think that's a feature of Flex atm)

Reply:
We could add the file to the `symfony/routing` recipe for 5.2. If we're on PHP 7, that file should have no effect. 🤔